src/main/java/com/tstepnik/planner/domain/Statistics.java

```diff
@@ -0,0 +1,39 @@
+package com.tstepnik.planner.domain;
+
+import lombok.*;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.time.LocalDateTime;
+
+@Getter
+@Setter
+@Entity
+@AllArgsConstructor
+@NoArgsConstructor
+public class Statistics {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private Long userId;
+
+    private Double productivity;
+
+    private Integer archivedTasksCount;
```

Comment:
Shouldn't our productivity be equal to finished*100/total (archived + finished, but not archived)? In this we are storing only archived. It introduces slight error to calculation but I'll let you decide if you want to change it.